web-extension/background.js

```diff
@@ -18,7 +19,7 @@ const interactor = new PopupClient()
 
 const encryptedStore = new EncryptedStorage(
   new ConcurrentStorage(new StorageLocalMap('wallets')),
-  process.env.REACT_APP_TEST
+  config.LIGHT_ENCRYPTION
```

Comment:
Instead of a feature flag, I'd probably prefer the actual settings in the config file :)